.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please don't commit this. Feel free to add .vscode/ to the .gitignore instead though

Reply:
Thank you, I will do so.